src/main/webapp/app/exam/participate/exam-participation.component.html

```diff
@@ -129,7 +129,11 @@
             (onExamEnded)="onExamEndConfirmed()"
             (onExamContinueAfterHandInEarly)="toggleHandInEarly()"
         ></jhi-exam-participation-cover>
-        <jhi-exam-participation-summary *ngIf="isOver() && studentExam.submitted" [studentExam]="studentExam"></jhi-exam-participation-summary>
+        <jhi-exam-participation-summary
+            *ngIf="isOver() && studentExam.submitted"
+            [studentExam]="studentExam"
+            [shouldFetchGradeInfoSeparately]="true"
```

Comment:
do we need this setting actually? Is it not always the case that the summary should fetch this information? I find it strange that the view determines if something should be downloaded from the server!

Reply:
Removed `shouldFetchGradeInfoSeparately` setting, now `jhi-exam-participation-summary` always fetches the grade info from server.